lib/Core/Tests/Input/JsonInputTest.php

```diff
@@ -66,8 +66,31 @@ public function provideSingleValuePairTests()
                                 'excluded-simple-values' => ['value3'],
                             ],
                         ],
+                        'order' => [
+                            'date' => 'uP',
+                        ],
                     ]
                 ),
+                ['@date' => 'ASC'],
+            ],
+            [
+                json_encode(
+                    [
+                        'fields' => [
+                            'name' => [
+                                'simple-values' => ['value', 'value2', '٤٤٤٦٥٤٦٠٠', '30', '30L'],
+                                'excluded-simple-values' => ['value3'],
+                            ],
+                        ],
+                    ]
+                ),
+                ['@date' => 'DESC', '@id' => 'ASC'],
+            ],
+            [
+                json_encode(
+                    []
+                ),
+                ['@date' => 'DESC', '@id' => 'ASC'],
```

Comment:
Default sort options still get applied, even if the search & filter part is empty.